src/main/java/org/opentripplanner/updater/package.md

```diff
@@ -0,0 +1,104 @@
+# Realtime Updaters
+
+## Realtime Data Sources
+
+Published transit data is broadly divided into two categories, which represent different time scales. On one hand we have scheduled data (also called static or theoretical data), and on the other hand realtime (or dynamic) data. Scheduled data is supplied in GTFS or NeTEx format, with the corresponding realtime data supplied in the [GTFS-RT](https://gtfs.org/realtime/reference/) and [SIRI](https://www.siri-cen.eu/) formats, respectively. This package contains code that retrieves and decodes realtime data, then layers it on top of the static transit data in a live OTP instance while it continues to handle routing requests.
+
+Different data producers might update their scheduled data every month, week, or day. Realtime data then covers any changes to service at a timescale shorter than that of a given producer's scheduled data. Broadly speaking, realtime data represents short-term unexpected or unplanned changes that modify the planned schedules, and could require changes to journeys that the riders would not expect from the schedule. OpenTripPlanner uses three main categories of realtime data which are summarized in the table below. The SIRI specification includes more types, but OTP handles only these three that correspond to the three GTFS-RT types.
+
+| GTFS-RT Name                                                                     | SIRI Name                | Description                                                           |
+|----------------------------------------------------------------------------------|--------------------------|-----------------------------------------------------------------------|
+| [Service Alert](https://gtfs.org/realtime/reference/#message-alert)              | Situation Exchange (SX)  | Text descriptions relevant to riders using a particular route or stop |
+| [Trip Update](https://gtfs.org/realtime/reference/#message-tripupdate)           | Estimated Timetable (ET) | Observed or expected arrival and departure times for near-term trips  |
+| [Vehicle Position](https://gtfs.org/realtime/reference/#message-vehicleposition) | Vehicle Monitoring (VM)  | Physical location of vehicles currently providing service             |
+
+GTFS-RT takes the form of binary protocol buffer messages that are typically fetched over HTTP. On the other hand, the SIRI specification originally described SOAP remote procedure calls retrieving an XML representation of the messages. Various projects instead adopted simple HTTP GET requests passing parameters in the URL and returning a JSON representation. The latter approach was eventually officially recognized as "SIRI Lite".
+
```

Comment:
If someone looks at CEN or third-party documentation and presentations on the specs, they often prominently mention that SIRI uses SOAP and XML. I am not fully able to verify this because as far as I know the docs still cost hundreds of Euros, so I'm digging through examples in old emails. But I believe SOAP-XML is still mentioned as the "main" or "serious" standard, official way to do SIRI. SIRI-Lite may be the more common in practice, but I think it's treated like an optional supplement to the spec. If OTP does something different than what the reader may have learned from official docs, it seems helpful to explain this background.

I don't think OTP documentation should be written as if only REST transport exists, it should acknowledge that this is a choice over the other method.

Reply:
I reworded it a bit to say SIRI Lite has become quite common and is the approach OTP supports.